src/pages/Admin/Community/CommunityListPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC, useMemo } from 'react';
+import { useRouteMatch } from 'react-router-dom';
+import { ListPage } from '../../../components/Admin';
+import { Loading } from '../../../components/core';
+import { useDeleteUserGroup } from '../../../hooks';
+import { useCommunityGroupsQuery } from '../../../hooks/generated/graphql';
+import { PageProps } from '../../common';
+
+interface CommunityListPageProps extends PageProps {
+  communityId: string;
+}
+
+export const CommunityListPage: FC<CommunityListPageProps> = ({ paths, communityId }) => {
+  const { url } = useRouteMatch();
+  const { data, loading } = useCommunityGroupsQuery({
```

Comment:
community query will be removed in the future

Reply:
Replaced with the new ecoverse.community query